source/StigData/Processed/VMW-vSphere-7-0-1.2.xml

```diff
@@ -0,0 +1,1352 @@
+      <DuplicateOf />
+      <IsNullOrEmpty>False</IsNullOrEmpty>
+      <LegacyId>
+      </LegacyId>
+      <OrganizationValueRequired>False</OrganizationValueRequired>
+      <OrganizationValueTestString />
+      <RawString>From the vSphere Client, go to Host and Clusters.
+
+Select a vCenter Server &gt;&gt; Configure &gt;&gt; Settings &gt;&gt; Advanced Settings. 
+
+Verify "VirtualCenter.VimPasswordExpirationInDays" is set to "30".
+
+or
+
+From a PowerCLI command prompt while connected to the vCenter server, run the following command:
+
+Get-AdvancedSetting -Entity &lt;vcenter server name&gt; -Name VirtualCenter.VimPasswordExpirationInDays
+
+If the "VirtualCenter.VimPasswordExpirationInDays" is set to a value other than "30" or does not exist, this is a finding.</RawString>
+    </Rule>
+    <Rule id="V-256356" severity="medium" conversionstatus="pass" title="SRG-APP-000516" dscresource="VMHostAdvancedSettings">
+      <AdvancedSettings>'' = '32'</AdvancedSettings>
+      <Description>&lt;VulnDiscussion&gt;The "vpxuser" password default length is 32 characters. Ensure this setting meets site policies; if not, configure to meet password length policies.
+
+Longer passwords make brute-force password attacks more difficult. The "vpxuser" password is added by vCenter, meaning no manual intervention is normally required. The "vpxuser" password length must never be modified to less than the default length of 32 characters.&lt;/VulnDiscussion&gt;&lt;FalsePositives&gt;&lt;/FalsePositives&gt;&lt;FalseNegatives&gt;&lt;/FalseNegatives&gt;&lt;Documentable&gt;false&lt;/Documentable&gt;&lt;Mitigations&gt;&lt;/Mitigations&gt;&lt;SeverityOverrideGuidance&gt;&lt;/SeverityOverrideGuidance&gt;&lt;PotentialImpacts&gt;&lt;/PotentialImpacts&gt;&lt;ThirdPartyTools&gt;&lt;/ThirdPartyTools&gt;&lt;MitigationControl&gt;&lt;/MitigationControl&gt;&lt;Responsibility&gt;&lt;/Responsibility&gt;&lt;IAControls&gt;&lt;/IAControls&gt;</Description>
+      <DuplicateOf />
+      <IsNullOrEmpty>False</IsNullOrEmpty>
+      <LegacyId>
+      </LegacyId>
+      <OrganizationValueRequired>False</OrganizationValueRequired>
+      <OrganizationValueTestString />
+      <RawString>From the vSphere Client, go to Host and Clusters.
+
+Select a vCenter Server &gt;&gt; Configure &gt;&gt; Settings &gt;&gt; Advanced Settings. 
+
+Verify "config.vpxd.hostPasswordLength" is set to "32".
+
+or
+
+From a PowerCLI command prompt while connected to the vCenter server, run the following command:
+
+Get-AdvancedSetting -Entity &lt;vcenter server name&gt; -Name config.vpxd.hostPasswordLength and verify it is set to 32.
+
+If the "config.vpxd.hostPasswordLength" is set to a value other than "32", this is a finding.
+
+If the setting does not exist, this is not a finding.</RawString>
+    </Rule>
+    <Rule id="V-256360" severity="medium" conversionstatus="pass" title="SRG-APP-000358" dscresource="VMHostAdvancedSettings">
+      <AdvancedSettings>'' = 'true'</AdvancedSettings>
```

Comment:
this setting is not correct- according to the rule this should be 'vpxd.event.syslog.enabled' = 'true'

Reply:
Corrections Made